code/modules/mob/living/carbon/xenomorph/castes/Lurker.dm

```diff
@@ -84,7 +84,7 @@
 		target_carbon.set_effect(get_xeno_stun_duration(target_carbon, 3), SUPERSLOW)
 		next_slash_buffed = FALSE
 		var/datum/action/xeno_action/onclick/lurker_assassinate/ability = get_xeno_action_by_type(bound_xeno, /datum/action/xeno_action/onclick/lurker_assassinate)
-		if (ability && istype(ability))
```

Comment:
This is already typechecked in `get_xeno_action_by_type()` so this should be redundant.

https://github.com/cmss13-devs/cmss13/blob/66e955747d0b18477465c8e813065c185d38ac0e/code/modules/mob/living/carbon/xenomorph/abilities/xeno_action.dm#L367-L374